server/src/main/java/com/thoughtworks/go/server/service/ArtifactStoreService.java

```diff
@@ -51,15 +51,15 @@ public ArtifactStores getPluginProfiles() {
     }
 
     public void create(Username currentUser, ArtifactStore newArtifactStore, LocalizedOperationResult result) {
-        update(currentUser, newArtifactStore, result, new CreateArtifactStoreConfigCommand(goConfigService, newArtifactStore, artifactExtension, currentUser, result));
+        update(currentUser, newArtifactStore, result, new CreateArtifactStoreConfigCommand(goConfigService, newArtifactStore, artifactExtension, currentUser, result), true);
```

Comment:
I think command should tell does it requires validation or on instead of passing it to `update(...)`